package.json

```diff
@@ -7,28 +7,28 @@
   "license": "MIT",
   "dependencies": {
     "bower": "~1.4.1",
-    "grunt": "~0.4.5"
-  },
-  "devDependencies": {
-    "connect-livereload": "~0.5.3",
     "connect-modrewrite": "~0.7.11",
-    "coveralls": "~2.11.2",
-    "grunt-bump": "~0.3.0",
+    "grunt": "~0.4.5",
     "grunt-contrib-clean": "~0.6.0",
```

Comment:
I agree but to be able to use `npm install --production` to selectively install I needed to move the dependencies required to run our `grunt build` task into `dependencies`. Otherwise we are required to install everything, inc protractor etc into the container just to run the build task. An alternative approach might be to run the grunt build in circle and not in the dockerfile, then we would not need all these dependencies installed.